src/inc/til/env.h

```diff
@@ -366,12 +366,13 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
             if (til::compare_ordinal_insensitive(var, temp) == 0 ||
                 til::compare_ordinal_insensitive(var, tmp) == 0)
             {
-                return til::details::wil_env::GetShortPathNameW<std::wstring, 256>(value.data());
-            }
-            else
-            {
-                return std::wstring{ value };
+                try
+                {
+                    return til::details::wil_env::GetShortPathNameW<std::wstring, 256>(value.data());

```

Comment:
Why not use the existing HRESULT variant?

Reply:
because it doesn't return a string >_<